typescript/infra/test/core.test.ts

```diff
@@ -60,6 +76,21 @@ describe('core', async () => {
     await AbacusCoreDeployer.transferOwnership(core, owners, multiProvider);
   });
 
+  it('persists partial failure', async () => {
+    try {
+      await flakeyDeployer.deploy();
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
+    expect(flakeyDeployer.deployedContracts).to.have.keys(['test1', 'test2']);
+  });
+
+  it('can be resumed from partial failure', async () => {
+    const result = await deployer.deploy(
+      flakeyDeployer.deployedContracts as any,
```

Comment:
Is this testing what you want it to test? In this test case, `flakeyDeployer.deployedContracts` will be empty no?

Reply:
good catch, it is indeed not testing what I want